src/sa11y.js

```diff
@@ -300,7 +300,7 @@ jQuery.noConflict();
 
             // Readability
             if (typeof sa11yReadabilityRoot !== "string" || $(sa11yReadabilityRoot).length === 0) {
-                sa11yReadabilityRoot = "main, [role='main']";
+                sa11yReadabilityRoot = "body";
```

Comment:
This change means that sa11y-english will need updating as well

https://github.com/ryersondmp/sa11y/blob/1931c3b2a67f2e76f64bcd1279b503f17763d0fc/src/sa11y-english.js#L9

Reply:
Good catch, @brianteeman - i'll fix comments.

Reply:
not sure the change is good though. why check readability on everything and not just main (by default)

seems to go against the objective of  Sa11y exclusively highlighting content issues.

Reply:
I agree once configured it should only scan "main". 

When I did some testing with bookmarklet version, I noticed that not all websites have a main content area. Secondly, since bookmarklet version scans entire page - one might assume it checks readability of entire page too. I don't think this is too big of a problem given that readability is only calculated from <p> and <li> tags only. If anything, I should perhaps add an exclusion for <li> tags within <nav>, <aside>, etc...

I'll consider reverting it back to <main> content area only.